crates/recent_projects/src/recent_projects.rs

```diff
@@ -235,13 +235,16 @@ impl PickerDelegate for RecentProjectsDelegate {
                 .inset(true)
                 .spacing(ListItemSpacing::Sparse)
                 .selected(selected)
-                .child(
-                    v_flex()
-                        .child(highlighted_location.names)
-                        .when(self.render_paths, |this| {
-                            this.children(highlighted_location.paths)
-                        }),
-                ),
+                .child(h_flex().gap_2().child(highlighted_location.names).when(
+                    self.render_paths,
+                    |this| {
+                        this.children(highlighted_location.paths.into_iter().map(|path| {
```

Comment:
Can there actually be multiple paths for an entry? I didn't find a case where that is happening. If it can actually happen this might look bad, because the paths are laid out horizontally.

Reply:
Aha thanks, I thought I was missing something there. I never used the "add folder to workspace" feature. I will try to make some improvements then.

Reply:
Yeah, though the multiple paths only show up if your most recent path (the top one) has multiple paths.